README.md

```diff
@@ -1,57 +1,21 @@
-# Project Name
+# InceptionV3 Model in Azure AI Gallery
 
-(short, 1-3 sentenced, description of the project)
+## Intro
+This repository contains code which demonstrates how to deploy and use an Inception model in Azure ML.
 
-## Features
+## How to Use
+1. [![Azure Notebooks](https://notebooks.azure.com/launch.png)](https://notebooks.azure.com/import/gh/gogowings/AML-Gallery-InceptionV3)
+[Import sample notebooks ](https://notebooks.azure.com/import/gh/gogowings/AML-Gallery-InceptionV3) into Azure Notebooks.
```

Comment:
Change it to https://notebooks.azure.com/import/gh/Azure-Samples/AML-Gallery-InceptionV3.